### PR TITLE
Space added in the template name documentation

### PR DESCRIPTION
**Pull request by iamajunravi**

### CODE REVIEW

**Review by Nixon-Samuel — APPROVED** [review]

File: docs/server-side/v2/legacy-api (v1)/room.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ You can create room using API. This provides a more scalable way of creating roo
 
 <PostRequest
     title="Create Room API"
-    desp="This endpoint is used to create Room. "
+    desp="This endpoint is used to create room. "
```
Comment (arjunattam) [inline docs/server-side/v2/legacy-api (v1)/room.mdx:14]: this only changes in the legacy API docs?

Comment (Nixon-Samuel) [reply docs/server-side/v2/legacy-api (v1)/room.mdx:14]: the casing was incorrect in legacy docs, hence updated it. But the changes are for the latest API docs. 
CC: @iamajunravi

Comment (arjunattam) [reply docs/server-side/v2/legacy-api (v1)/room.mdx:14]: they are not updated in this PR?

Comment (arjunattam) [reply docs/server-side/v2/legacy-api (v1)/room.mdx:14]: my bad - i thought room names were supporting spaces. but we have updated on template names

**Review by arjunattam — APPROVED** [review]

File: docs/server-side/v2/legacy-api (v1)/room.mdx (hunk 1 above)
Comment (Nixon-Samuel) [reply docs/server-side/v2/legacy-api (v1)/room.mdx:14]: 👍

**Pull request merged**